nym-node-status-api/src/http/api/testruns.rs

```diff
@@ -23,31 +24,32 @@ pub(crate) fn routes() -> Router<AppState> {
     Router::new()
         .route("/", axum::routing::get(request_testrun))
         .route("/:testrun_id", axum::routing::post(submit_testrun))
+        .layer(DefaultBodyLimit::max(1024 * 1024 * 5))
```

Comment:
any reason for explicitly adding it? were we accidentally ddosing ourselves? ; D

Reply:
this is request size limit

I was hitting it when probe log level was set to trace so I don't want the API to break if someone intentionally does that

and it's added ONLY here so other endpoints are protected by a [lower default (2MB)](https://docs.rs/axum/latest/axum/extract/struct.DefaultBodyLimit.html)